### PR TITLE
[FrameworkBundle] Remove doctrine/annotations integration

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 7.0
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | Follows #50888
| License       | MIT
| Doc PR        | TODO, see symfony/symfony-docs#18589

### CODE REVIEW

File: UPGRADE-7.0.md [hunk 1]
```diff
@@ -121,6 +121,8 @@ FrameworkBundle
    $this->render(..., ['form' => $form]);
    ```
 
+ * Remove the integration of Doctrine annotations
```
Comment (nicolas-grekas) [inline UPGRADE-7.0.md]:
```suggestion
 * Remove the integration of Doctrine annotations, use native attributes instead
```

File: src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ CHANGELOG
    `Symfony\Component\Serializer\Normalizer\PropertyNormalizer` autowiring aliases, type-hint against
    `Symfony\Component\Serializer\Normalizer\NormalizerInterface` or implement `NormalizerAwareInterface` instead
  * Remove the `Http\Client\HttpClient` service, use `Psr\Http\Client\ClientInterface` instead
+ * Remove the integration of Doctrine annotations
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md]:
```suggestion
 * Remove the integration of Doctrine annotations, use native attributes instead
```

**Review by nicolas-grekas — COMMENTED** [review]
please mind fabbot :)

File: UPGRADE-7.0.md [hunk 3]
```diff
@@ -121,6 +121,8 @@ FrameworkBundle
    $this->render(..., ['form' => $form]);
    ```
 
+ * Remove the integration of Doctrine annotations, use native attributes instead
```
Comment (Seb33300) [inline UPGRADE-7.0.md:124]: This was not supposed to be removed in Symfony 8?

Comment (derrabus) [reply UPGRADE-7.0.md:124]: Nope.

Comment (Seb33300) [reply UPGRADE-7.0.md:124]: In https://github.com/symfony/symfony/pull/50888, you said:
> The plan is to make this the default and only valid option in Symfony 7 and remove that setting entirely in Symfony 8.

Comment (Seb33300) [reply UPGRADE-7.0.md:124]: Ohh sorry, just got it. The option cannot be enabled on Symfony 7

Comment (derrabus) [reply UPGRADE-7.0.md:124]: Yes, the setting is still there. You just cannot enable it.

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Pull request merged**